sdk/inc/azure/iot/az_iot_common.h

```diff
@@ -166,7 +166,7 @@ AZ_NODISCARD az_result az_iot_message_properties_find(
  * @param[out] out_value A pointer to an #az_span containing the value of the next property.
  * @return An #az_result value indicating the result of the operation.
  * @retval #AZ_OK A property was retrieved successfully.
- * @retval #AZ_ERROR_IOT_NO_MORE_PROPERTIES The API reached the end of the properties to retrieve.
+ * @retval #AZ_ERROR_IOT_END_OF_PROPERTIES The API reached the end of the properties to retrieve.
```

Comment:
Update az_result.h

Reply:
Yea I don't know what happened....somehow find all and replace didn't like the actual place it mattered most.

Reply:
VS or VSCode? I have noticed, whenever I have done a rename (instead of find/replace) in VS, it doesn't update the header file for me, but updates the rest.

![image](https://user-images.githubusercontent.com/6527137/91605068-b6038580-e924-11ea-815a-3def9aac3362.png)